### ISSUE TITLE
switchToLatest bug allusion

### ISSUE THREAD

**Issue by moreindirection** [mattneub/understandingCombine#11]
https://github.com/mattneub/understandingCombine/blob/236c4af76689d71d985cd00bf64a1b726101ca35/operators/operatorsTransformersBlockers/operatorsSwitchToLatest.txt#L32

Would be interesting to know more about this bug.

**Comment by mattneub** [comment]
I don't think it would, at this point. But I do vaguely remember they just hadn't worked out the details correctly.
